public/content-script.js

```diff
@@ -0,0 +1,140 @@
+// разработка попап
+
+const takeIt = document.createElement('div');
+
+//разработка логотипа с стилями
```

Comment:
It would be better to structure code with separate functions for example logoPopup = () => { const logo ...}